supabase/migrations/20260826110000_lock_r2_direct_ota_versions.sql

```
@@ -0,0 +1,462 @@
+-- GHSA-5rg9-rhwj-wj76: r2-direct staging rows were treated as "not ready" and
+-- stayed mutable via PostgREST after checksum/session_key were set. Upload is
+-- complete once storage_provider is no longer r2-direct; during r2-direct staging
```

SQLFluff (4.3.0)

[error] Line is too long (82 > 80).

(LT05)